### DISCUSSION TITLE
Arrow Icon Meanings on the Holdings page

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [ghostfolio/ghostfolio#2320] [Q&A]
This is probably trival...but what do the arrow icons to the left of the holding name on the holdings page mean?  I understand the green arrow up and the red arrow down...but there's the gray arrow flat to the right (I assume this means no-change), but then I also see a gray pause like icon...what do they mean?

All of my investments are US based mutual funds, many if not most of mine have the Pause button next to them.  

![image](https://github.com/ghostfolio/ghostfolio/assets/68966864/ca08091d-484d-415a-8d3d-283264ee8463)

**Comment by dtslvr** [comment]
Hi @htims05

The pause symbol indicates that the market is currently closed according to the data provider.

**Reply by Lunchtime0614** [reply]
Humm either it’s not working correctly or YAHOO is not producing the proper data…for example right now at 6:45 am eastern time half of my mutual funds show a red or green arrow and the other half show the pause icon…they are all fidelity mutual funds. 

Is there a way I can manually check this info on YAHOO directly?
